index.js

```diff
@@ -3,8 +3,12 @@
 var path = require('path');
 var isglob = require('is-glob');
 var pathDirname = require('path-dirname');
+var isWin32 = require('os').platform() === 'win32';
 
 module.exports = function globParent(str) {
+	// flip windows path separators
+	if (isWin32 && str.indexOf('/') < 0) str = str.split('\\').join('/');
```

Comment:
> won't be passing the glob parent

to clarify, I meant "won't be passing glob parent as a glob pattern". sorry that was vague